src/background/services/monetization.ts

```diff
@@ -259,6 +265,20 @@ export class MonetizationService {
     })
   }
 
+  private onKeyRevoked() {
+    this.events.once('open_payments.key_revoked', async () => {
+      this.logger.warn(`Key revoked. Stopping all payment sessions.`)
+      for (const sessions of Object.values(this.sessions)) {
+        for (const session of sessions.values()) {
+          session.stop()
+        }
+      }
```

Comment:
Do we want to stop all or just active tab(s) ones?
My concern here is how to works in Edge's split screen view.

Reply:
Hmm, thanks for pointing out Edge's split screen view. I'm okay with the implementation as is, but we need to spend some quality time on Edge as well to check how this would work. Let's create an issue for this.

Reply:
This loop would need to be modified regardless in https://github.com/interledger/web-monetization-extension/issues/381 if we store tabs in a WeakMap (Weakmaps can't be iterated, so we'd query active tabs and then iterate over that).